packages/react/tooltip/src/Tooltip.tsx

```diff
@@ -231,9 +259,12 @@ const TooltipTrigger = React.forwardRef<TooltipTriggerElement, TooltipTriggerPro
   (props: ScopedProps<TooltipTriggerProps>, forwardedRef) => {
     const { __scopeTooltip, ...triggerProps } = props;
     const context = useTooltipContext(TRIGGER_NAME, __scopeTooltip);
+    const providerContext = useTooltipProviderContext(TRIGGER_NAME, __scopeTooltip);
     const popperScope = usePopperScope(__scopeTooltip);
-    const composedTriggerRef = useComposedRefs(forwardedRef, context.onTriggerChange);
+    const ref = React.useRef<TooltipTriggerElement>(null);
+    const composedRefs = useComposedRefs(forwardedRef, ref, context.onTriggerChange);
     const isPointerDownRef = React.useRef(false);
+    const hasTriggerActivated = React.useRef(false);
```

Comment:
What does this one mean?

When I read the name I thought it was about the trigger activation (click, keydown space/enter) but looking at where the ref is updated it seems to mean something else.

Reply:
We needed to use `pointerMove` after all for the polygon bleed change. Because of this we only want to trigger an open on the very first `pointerMove` event, this flag is to describe that the tooltip "has opened" already so don't do it again until we leave and come back.

Perhaps something like `hasPointerMoveOpened`?

Reply:
Yeah that makes more sense.